### PR TITLE
GUI-2894: No need to pass token for delete request

### PR DESCRIPTION
**Pull request by kamalgill**
https://eucalyptus.atlassian.net/browse/GUI-2894

### CODE REVIEW

File: eucaconsole/static/js/pages/bucket_details.js [hunk 1]
```diff
@@ -97,8 +97,7 @@ angular.module('BucketDetailsPage', ['S3SharingPanel', 'EucaConsoleUtils', 'Cors
             $scope.deleteError = '';
             $scope.deletingCorsConfig = true;
             var deleteDialog = $('#cors-delete-confirmation-modal');
-            var csrfToken = angular.element('#csrf_token').val();
-            CorsService.deleteCorsConfig($scope.bucketName, csrfToken)
+            CorsService.deleteCorsConfig($scope.bucketName)
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/bucket_details.js]: I think you should keep csrf, no? This is a modify operation. Look here:
https://www.owasp.org/index.php/REST_Security_Cheat_Sheet#Protect_HTTP_methods
"For resources exposed by RESTful web services, it's important to make sure any PUT, POST, and DELETE request is protected from Cross Site Request Forgery. Typically one would use a token-based approach. See Cross-Site Request Forgery (CSRF) Prevention Cheat Sheet for more information on how to implement CSRF-protection."

Comment (kamalgill) [inline eucaconsole/static/js/pages/bucket_details.js]: @dkavanagh : Please review the comments in the ticket re why we removed it.

Comment (dkavanagh) [inline eucaconsole/static/js/pages/bucket_details.js]: Not clear to me. Let's talk, perhaps involving Vika.

**Comment by kamalgill** [comment]
Closing w/o merging -- replacement PR forthcoming.

**Pull request closed without merging**